shared/src/question-generators/recursion/formulaUtils.test.ts

```diff
@@ -6,60 +6,84 @@ describe("parseRecursiveFunction", () => {
   test("Handles T(n/3) + 7", () => {
     expect(parseRecursiveFunction("T(n/3) + 7")).toEqual({
       a: 1,
+      // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
So `T` does not follow the naming convention. But since there is no `t` in the the code anywhere, `T` should just be named `t` and so the eslint comments are not needed anymore. As a side note: such one-letter names are generally bad (with a few rare exceptions) and should be avoided but since they correspond to the formulas its is fine.